### ISSUE TITLE
Terms and Term Limits for Chairs/TLs

### ISSUE THREAD

**Issue by parispittman** [kubernetes/community#5886]
Following up from Steering Committee meeting July 12th, we are going to split issue #5855 into three parts:
1- the original issue #5855: mechanism for selection, 
2- defining terms and any limits associated (this issue), 
3- should 1 tech lead be a required role for each SIG in addition to 1 chair

------

Chairs should have terms and potentially limits. 

Why: 
1. Responsiveness of overburdened chairs #4289 who have multiple roles and "hats" within the project (several related linked issues inside of that issue that would be related)
2. No current forcing function to think about succession or do outreach for new chairs and/or tech leads
3. Will help build a leadership pipeline that feeds into multiple areas of the project including Steering
4. New people bring fresh ideas and help us find holes that helps with sustainability 
5. [will update]

Knowns:
- [data placeholder for how many explicit tech leads we have and combo chair/tl]
- [data placeholder for average time served and other time metrics]

Open Questions:
1 What's a term defined as?
2 How long should a term be?
3 Option to renew? / extend? / eligibility for same role in another part of the project?

Proposed solution(s):
- will collect discussion themes here

**Comment by ehashman** [comment]
Can we consider splitting

>  2- defining terms and any limits associated (this issue),

into 

>  2- defining terms
>  3- defining any term limits associated

and perhaps move term limits into a separate GH issue? Based on the discussion in the Tuesday July Chairs/Leads call, it was clear to me that many people view them as separate issues.

(Also, should we use GH discussions rather than issues?)

**Comment by derekwaynecarr** [comment]
if we establish terms, and term limits, we will need to determine if we need to set minimum requirements for a chair or TL to not have a min=1 participation list.  i suspect we will want to support rotation of senior/junior folks by requiring some min=2 for either role.

**Comment by deads2k** [comment]
> 4 should the scope of this issue be just chairs or chairs and TLs?

I don't think that term limits are appropriate for tech leads.  Technical knowledge and merit doesn't have a time limit. If a person is willing to continue performing that role, excluding them simply because they've done it before doesn't seem appropriate or efficient.

I feel slightly less strongly about that for chairs, since the chair role requires less technical depth with the project.  However, I think that if someone performs the role effectively and is interested in continuing, I would allow them to remain eligible.

**Comment by deads2k** [comment]
> support rotation of senior/junior folks

@derekwaynecarr When you say junior with respect to tech leads, do you mean that as "aspiring to be a tech lead, but lacking enough experience and depth to do it effectively" or do you mean that as, "someone eligible for the position on merit (not aspirational), but less experience that the other tech lead they're working with"?

**Comment by lavalamp** [comment]
> avoid BDFL in certain areas

IMO benevolent dictators are great; the actual problem is a) inability to determine benevolence in advance combined with b) no clear mechanism for ejecting (or even detecting) non- or formerly-benevolent dictators.

**Comment by derekwaynecarr** [comment]
@deads2k I was just trying to capture that I think term limits requires more than 1 concurrent individual to be effective.  So senior/junior just meant the persons that were staggered based on when they held a role similar to how steering for example staggers its own elections.  It was not intended to identify competency or merit.

**Comment by lavalamp** [comment]
My position: additional policy is harmful unless proven otherwise. I feel like there hasn't been a good discussion about the *problem* (with concrete specifics) and that makes it hard to see if the proposed solution actually addresses it. I am sure no one wants to say bad things about any specific SIG, but I don't see how we can all come to agreement that there is a problem at all without some kind of specific evidence like this.

**Comment by jberkus** [comment]
other advantages:

- having a defined interval at which we talk about replacing chairs would mean that chairs could leave *and come back*

I am -1 to term limits at this point.
